Make the ppolicy overlay call udm to lock an account (Bug #31907)

diff --git debian/control debian/control
--- debian/control
+++ debian/control
@@ -19,6 +19,7 @@ Build-Depends: debhelper (>= 10),
                nettle-dev <!stage1>,
                perl:any,
                po-debconf,
+               python-dev,
                unixodbc-dev <!stage1>
 Build-Conflicts: libbind-dev, bind-dev, libicu-dev, autoconf2.13
 Standards-Version: 4.3.0
@@ -31,6 +32,7 @@
 Pre-Depends: debconf (>= 0.5) | debconf-2.0, ${misc:Pre-Depends}
 Depends: ${shlibs:Depends}, libldap-2.4-2 (= ${binary:Version}),
  univention-config,
+ python-univention-directory-manager (>=10.0.15-5),
  coreutils (>= 4.5.1-1), psmisc, perl (>> 5.8.0) | libmime-base64-perl,
  adduser, lsb-base (>= 3.2-13), ${misc:Depends}
 Recommends: libsasl2-modules
